src/Tools/Source/RunTests/ProcessTestExecutor.cs

```diff
@@ -91,7 +101,10 @@ private string GetResultsFilePath(AssemblyInfo assemblyInfo)
                     lowPriority: false,
                     displayWindow: false,
                     captureOutput: true,
-                    cancellationToken: cancellationToken);
+                    cancellationToken: cancellationToken,
+                    environmentVariables: environmentVariables,
+                    onProcessStartHandler: (process) => ProcDumpRunner.StartProcDump(process.Id, process.ProcessName, outputDirectory, Logger.Log)
```

Comment:
There is a race condition here by which the process could exit before procdump strats. What will happen in that case?